packaging/debian-unstable/copyright

```diff
@@ -0,0 +1,22 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: snappy
```

Comment:
Again, fix in Ubuntu side first pls?

BTW, debian/copyright in Ubuntu (but not Debian) really should list the copyright info for everything in vendor/...